pages/public_cloud/containers_orchestration/managed_kubernetes/forwarding-audit-logs-to-logs-data-platform/guide.en-gb.md

```diff
@@ -0,0 +1,197 @@
+
+###### 1/ Retrieve your target Stream (and ID):
+
+List data stream of you Logs Data Platform account ([API reference](https://eu.api.ovh.com/console-preview/?section=%2Fdbaas%2Flogs&branch=v1#get-/dbaas/logs/-serviceName-/output/graylog/stream)):
+
+```shell
+GET /dbaas/logs/{serviceName}/output/graylog/stream
+```
+Get the details of a data stream ([API reference](https://eu.api.ovh.com/console-preview/?section=%2Fdbaas%2Flogs&branch=v1#get-/dbaas/logs/-serviceName-/output/graylog/stream/-streamId-)):
+```shell
+GET   /dbaas/logs/{serviceName}/output/graylog/stream/{streamId}
+```   
+###### 2/ Create your subscription
+
+As per below example, the POST request has a payload containing a streamId, which is the target data stream of your LDP account where you aim your Kubernetes cluster Audit logs will be forwarded. You also need to specifiy the 'kind' of log you want to forward, note that currenlty the only supported value for Managed Kubernetes Service is 'audit' (you can find available kinds using the [dedicated API](https://eu.api.ovh.com/console-preview/?section=%2Fcloud&branch=v1#get-/cloud/project/-serviceName-/capabilities/kube/log/kind) call) :
+```shell
+POST /cloud/project/{serviceName}/kube/{kubeId}/log/subscription
+{
+  "kind": "string",   //Currently the only supported value is 'audit'
+  "streamId": "18d602ec-af40-4000-8e59-41ecc8c23f80" //the streamID of the targeted Stream
+}
+```
+
+You will get in response an operationId
+```shell
+{
+"operationId": "18d3aecd-bae0-4000-8558-4b60e496bb80",
+"serviceName": "string"
+}
+```
+
+So you can use it to retrieve the subscriptionid further management purposes using Logs Data Platform read operation endpoint:
+```shell
+GET /cloud/project/{serviceName}/kube/{kubeId}/log/subscription/{subscriptionId}
+{
+  "createdAt": "2024-01-31T15:45:25.286Z",
+  "kind": "string",
+  "resource": {
+    "name": "string",
+    "type": "string"
+  },
+  "serviceName": "string",
+  "streamId": "string",
+  "subscriptionId": "18d60324-b260-4000-83db-b484f4db6e80",
+  "updatedAt": "2024-01-31T15:45:25.286Z"
+}
+```
+
+## How to use your Kubernetes Audit logs?
+Now that your Kubernetes instance Audit logs are ingested and stored in your Logs Data Platform data stream, you can query your logs and build dashboards to have a graphical representation of your logs using web-based UI of Graylog.
+-	On the OVHcloud Mananger retrieve the admin user and its password in your Logs Data Platform account home page.
+-	Open the Graylog web-ui. You can retrieve le link in your account home page or using your Access point depending of your account region (for example: Gravelines regions https://gra1.logs.ovh.com/ )
+-	Login Graylog using your Logs Data Platform service name and Password
+-	Search through your logs across the data stream of your Logs Data Platform account. You can refer to Graylog writing search queries documentation for details on search syntax.
```

Comment:
For "You can refer to Graylog writing search queries documentation for details on search syntax", you can link the official documentation: https://go2docs.graylog.org/4-x/making_sense_of_your_log_data/writing_search_queries.html

As we are "stuck" in Graylog v4.x, this link won't change

Reply:
```suggestion
-	Search through your logs across the data stream of your Logs Data Platform account. You can refer to [Graylog writing search queries documentation](https://go2docs.graylog.org/4-x/making_sense_of_your_log_data/writing_search_queries.html) for details on search syntax.
```